src/pages/en/guides/publish-to-npm.md

```diff
@@ -6,37 +6,42 @@ description: Learn how to publish Astro components to NPM
 
 Building a new Astro component? **Publish it to [npm!](https://npmjs.com/)**
 
-Publishing a component is a great way to reuse work across your team, your company, or the entire world. Astro components can be published to and installed from npm, just like any other JavaScript package.
+Publishing an Astro component is a great way to reuse your existing work across your projects, and to share with the wider Astro-community at large. Astro components can be published directly to and installed from NPM, just like any other JavaScript package.
```

Comment:
nit: `and to share your work with the wider Astro community at large` (I don't think you'd hyphenate Astro-community? tbh I'm not sure!)